src/server.ts

```diff
@@ -91,39 +89,71 @@ setInterval(() => {
 */
 
 const didConfig = '.well-known/did-configuration.json';
+
+type TimeSinceEpoch = number
+type DateTimeString = string
+type LongFormDID = string
+
+type DomainLinkageCredentialJWTPayload = {
+   exp: TimeSinceEpoch,
+   iss: LongFormDID 
+   nbf: TimeSinceEpoch,
+   sub: LongFormDID,
+   vc: {
+       "@context": [
+           "https://www.w3.org/2018/credentials/v1",
+           "https://identity.foundation/.well-known/did-configuration/v1"
+        ],
+        credentialSubject: {
+            id: LongFormDID,
+            origin: string
+        },
+        expirationDate: DateTimeString,
+        issuanceDate: DateTimeString,
+        issuer: LongFormDID,
+        type: ["VerifiableCredential", "DomainLinkageCredential"]
+   }
+}
+
+type WellKnownDidConfigurationResponse = {
+    "@context": "https://identity.foundation/.well-known/did-configuration/v1",
+    linked_dids: [string]
+}
 app.get('/'  + didConfig, async (req, res, err) => {
     try {
-        const issued = Math.round(new Date().getTime() / 1000 - 10 * 60); // ten minutes ago
-        const expires = Math.round(new Date().getTime() / 1000 + 10 * 60); // ten minutes from now
-        const response = {
-            "@context": "https://identity.foundation/.well-known/contexts/did-configuration-v0.0.jsonld",
-            "entries": [
-            await issuerState.sk.sign({
-                kid: issuerState.did + '#signing-key-1',
-                }, {
-                    sub: issuerState.did,
-                    iss: issuerState.did,
-                    nbf: issued,
-                    exp: expires,
-                    vc: {
-                        "@context": [
-                            "https://www.w3.org/2018/credentials/v1",
-                            "https://identity.foundation/.well-known/contexts/did-configuration-v0.0.jsonld"],
-                        issuer: issuerState.did,
-                        issuanceDate: new Date(issued * 1000).toISOString(),
-                        expirationDate: new Date(expires * 1000).toISOString(),
-                        type: [
-                            "VerifiableCredential",
-                            "DomainLinkageCredential"
-                        ],
-                        credentialSubject: {
-                            id: issuerState.did,
-                            origin: new URL(issuerState.config.serverBase).origin,
-                        }
-                    }
-                })
+        const issued_seconds_since_epoch = Math.round(new Date().getTime() / 1000 - 10 * 60) // ten minutes ago
+        const expires_seconds_since_epoch = Math.round(new Date().getTime() / 1000 + 10 * 60) // ten minutes from now
```

Comment:
Just a nit, but would be good to follow the camelCase naming convention used throughout the codebase. (I know, we're inserting a few of these things into JSON objects with snake_case property names, so there's always going to be something that's an awkward fit.)

Reply:
silly me – working in too many languages in a single day! Yeah, absolutely will change.

Reply:
No worries -- will wait until you get a chance to update these, and then I'm happy to merge. Thanks again.